File: pinyougou-manager-web/src/main/webapp/js/service/itemCatService.js
app.service('itemCatService',function($http){
    this.findByParentId = function (parentId) {
        return $http.get('../itemCat/findByParentId.do?parentId='+parentId);
    }
    this.deleteByIds = function (selectIds) {
        return $http.post('../itemCat/deleteByIds.do',selectIds);
    }
    this.save = function (entity) {
        return $http.post('../itemCat/save.do',entity);
    }
    this.findOne = function (id) {
        return $http.get('../itemCat/findOne.do?id='+id);
    }
});

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/ItemCatService.java
package com.pinyougou.sellergoods.service;

import com.pinyougou.pojo.TbItemCat;

import java.util.List;

public interface ItemCatService {
    List<TbItemCat> findByParentId(Long parenId);

    void deleteByIds(Long[] ids);

    void save(TbItemCat tbItemCat);

    TbItemCat findOne(Long id);
}
